assets/scss/base/_lists.scss

```diff
@@ -1,10 +1,21 @@
+/*
+Lists
 
-/**
-* @desc         lists.
-* @name         layouts
-* @author       Anthony Munene
-* @tested       N/A
-* @requires     N/A
+Unordered lists are used for representing data in a list format.
+
+Markup:
+<ul class="{{modifier_class}}">
+  <li>List item 1</li>
+  <li>List item 2</li>
+  <li>List item 3</li>
+</ul>
+
+.bullets                      - A bulleted list using the modifier `.bullets`
```

Comment:
A piece of work has been added to the component library in https://github.com/hmrc/component-library-template/commit/b2bff44f01ef7f3cdadb4727fdf3519105a897ff
This work takes the `modifier` and prepends it to the modifier description automatically. Therefore you don't need `.bullets` at the end as it is automatically prepended.

Please update the component library in you local checkout - details here https://github.com/hmrc/assets-frontend#updating-the-component-library-dependency